presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaSplitManager.java

```diff
@@ -178,31 +128,4 @@ private static boolean isURI(String location)
         }
         return true;
     }
-
-    private static long[] findAllOffsets(SimpleConsumer consumer, String topicName, int partitionId)
```

Comment:
Try to use `org.apache.kafka.clients.consumer.KafkaConsumer#beginningOffsets` to restore segments